configs/textrecog/nrtr/metafile.yml

```diff
@@ -6,7 +6,7 @@ Collections:
       - Adam
     Epochs: 6
     Batch Size: 6144
-    Training Resources: 1x Tesla A100
+    Training Resources: 4x NVIDIA A100-SXM4-80GB
```

Comment:
For all nrtr models, batch size = 384, Training Resources = 1x NVIDIA A100-SXM4-80GB